### PR TITLE
FI-3267: Upgrade US Core Test Kit to use smart-app-launch-test-kit v0.4.5

### PR DESCRIPTION
**Pull request by emichaud998**
# Summary
Upgrade US Core Test Kit to use smart-app-launch test kit version 0.4.5. Also updated the test kit to integrate SMART v2.2 into the test kit.

Note: Did not integrated SMART v2.2 changes that SMART v2.2 CORs test PR brings in. This will be a later ticket.

# Testing Guidance
Run locally and ensure everything works as expected

### CODE REVIEW

File: us_core_test_kit.gemspec [hunk 1]
```diff
@@ -9,9 +9,9 @@ Gem::Specification.new do |spec|
   spec.description   = 'US Core Inferno tests'
   spec.homepage      = 'https://github.com/inferno-framework/us-core-test-kit'
   spec.license       = 'Apache-2.0'
-  spec.add_runtime_dependency 'inferno_core', '>= 0.4.37'
-  spec.add_runtime_dependency 'smart_app_launch_test_kit', '>= 0.4.0'
-  spec.add_runtime_dependency 'tls_test_kit', '~> 0.2.0'
+  spec.add_dependency 'inferno_core', '>= 0.4.37'
```
Comment (yunwwang) [inline us_core_test_kit.gemspec:12]: Curious, why change add_rumtime_dependency to add_dependency?

Comment (emichaud998) [reply us_core_test_kit.gemspec:12]: Hmm it looks like my rubocop extension did that automatically. Would it be better if I changed it back to add_runtime_dependency?

Comment (yunwwang) [reply us_core_test_kit.gemspec:12]: LOL. `add_dependency` is the old name and `add_runtime_dependency` is the new name. Though they are the same, I think it is better to reverse the change to keep symmetry from the following `add_development_dependency` statement.

File: us_core_test_kit.gemspec [hunk 2]
```diff
@@ -9,9 +9,9 @@ Gem::Specification.new do |spec|
   spec.description   = 'US Core Inferno tests'
   spec.homepage      = 'https://github.com/inferno-framework/us-core-test-kit'
   spec.license       = 'Apache-2.0'
-  spec.add_runtime_dependency 'inferno_core', '>= 0.4.37'
-  spec.add_runtime_dependency 'smart_app_launch_test_kit', '>= 0.4.0'
-  spec.add_runtime_dependency 'tls_test_kit', '~> 0.2.0'
+  spec.add_development_dependency 'inferno_core', '>= 0.4.37'
```
Comment (yunwwang) [inline us_core_test_kit.gemspec:12]: Oh no. These three should be changed back to `spec.add_runtime_dependency`

**Review by yunwwang — APPROVED** [review]

**Pull request merged**